test/maxrows.test.js

```diff
@@ -26,7 +25,7 @@ describe('cloudant max rows', function() {
     });
     Thing.belongsTo('foo', {model: Foo});
     Foo.hasMany('things', {foreignKey: 'fooId'});
-    db.automigrate(done);
+    done();
```

Comment:
@kjdelisle there is another bug of `automigrate`: https://github.com/strongloop/loopback-connector-cloudant/issues/92

Could be related to https://github.com/strongloop/loopback-connector-cloudant/issues/94